src/avram/add_column_attributes.cr

```diff
@@ -0,0 +1,125 @@
+# This module creates methods for each column in a model that map
+# to an `Avram::Attribute` as well as methods that fill those attributes
+# with values that comes from params.
+module Avram::AddColumnAttributes
```

Comment:
This code was in both SaveOperation and DeleteOperation, but seemed to have a few slight differences. One being that it was generating a `permitted_params` method for every column, but that method never did anything at the compiler level. So this becomes a small refactor since I had to change how things are checked.